liquidprompt

```diff
@@ -4797,11 +4797,18 @@ _lp_default_theme_prompt() {
 # Construct the prompt #
 ########################
 
+_LP_NOT_A_REAL_COMMAND=0
 __lp_set_prompt() {
     # Display the return value of the last command, if different from zero
     # As this get the last returned code, it should be called first
     local -i lp_error="$?"
 
+    if (( _LP_NOT_A_REAL_COMMAND )); then
+        lp_error=0
+    fi
+    # zsh doesn't run __lp_before_command, so we set it here
```

Comment:
Bash also doesn't run `__lp_before_command()` if all the features that depend on it are disabled.